src/index.js

```diff
@@ -448,6 +482,139 @@ class DocumentReferencesVisualizer extends GenericVisualizer {
   };
 }
 
+class ServiceRequestVisualizer extends GenericVisualizer {
+  static defaultProps = {
+    title: 'Service Requests',
+    columns: [
+        { title: 'SNOMED', versions: '*', getter: s => s.code.coding[0].code },
```

Comment:
Seeing this makes me realize, this library was originally built specifically for Synthea patients, where certain resources can only have certain types (like Conditions are always SNOMED and never ICD-10, Medications are always RxNorm, etc) but that's not going to be the case for all FHIR. So I'm fine with this saying SNOMED but we can also make it a more generic "Code" column. Same with any other places we have a column title of SNOMED/LOINC/RxNorm.

Reply:
I ended up ditching all of the system specific column headers in the code/display formatting combination. Lemme know what you think!